.github/workflows/container-autoupdate.yml

```diff
@@ -36,6 +35,29 @@ jobs:
         with:
           ref: ${{ matrix.branch }}
 
+      # FIXME: Remove when crun 0.18 will be available by default, see issue https://github.com/containers/podman/issues/9442
+      - name: Install newest crun from source (workaround for ELN)
+        if: matrix.container-tag == 'eln'
+        run: |
+          set -eux
+          echo "Currently installed crun version:"
+          crun --version
```

Comment:
Written this way, we will always use 0.18. Please make it also fail if the pre-installed version is no longer 0.17:

```
CRUN_VERSION=$(crun --version | grep "^crun version 0.17$")
echo "$CRUN_VERSION"
[ -z "$CRUN_VERSION" ] && exit 17
```

You can insert ` | sed -e "s/0.17/0.18/"` before grep to test the failure mode.

Reply:
Alternatively, dismiss this and merge away, I can do this part as a follow-up PR.